gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +211,59 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def background_links(self, data_store, modeltype, out_dir_background_model=None, outfile=None,
```

Comment:
Still a problem on the docstring but I will fix this tomorow,
What do you think in general?